applogic/src/api/types.rs

```diff
@@ -358,12 +367,76 @@ impl From<NotificationType> for UiNotificationType {
             NotificationType::ConversationChange(conversation_id) => {
                 UiNotificationType::ConversationChange(conversation_id)
             }
-            NotificationType::Message(message) => UiNotificationType::Message(message.into()),
+            NotificationType::Message(message) => UiNotificationType::Message((*message).into()),
         }
     }
 }
 
-#[derive(Debug, Clone)]
+/// Position of a conversation message in a flight.
+///
+/// A flight is a sequence of messages that are grouped to be displayed together.
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Hash)]
+pub enum UiFlightPosition {
+    /// The message is the only message in the flight.
+    Unique,
```

Comment:
I know this was my suggestion, but it should be called `Single`.

Reply:
Sure, renamed.